README.md

```diff
@@ -65,3 +65,34 @@ This project adheres to the [StandardJS Style Guide](https://github.com/standard
 ## Entity-Relationship Diagram
 
 ![](./diagram.png)
+
+## Table Definitions
+
+```sql
+CREATE TABLE users(
+  id SERIAL PRIMARY KEY,
+  username varchar(32) UNIQUE NOT NULL,
+  email varchar(50) UNIQUE NOT NULL,
+  password text NOT NULL
+);
+
+CREATE TABLE communities(
+  id SERIAL PRIMARY KEY,
+  username varchar(32) UNIQUE NOT NULL
+);
+
+CREATE TABLE posts(
+  id SERIAL PRIMARY KEY,
+  id integer REFERENCES users NOT NULL,
```

Comment:
I know this is a valid shorthand to reference the `id` column in `users`, but is it valid to have more than 1 `id` column in a single table? I don't think so but can't test at the moment, can you?

Reply:
Go with user_id on line 86